lib/aur-build

```diff
@@ -397,8 +396,11 @@ while IFS= read "${read_args[@]}" -ru "$fd" path; do
 
     if (( create_package )); then
         if (( chroot )); then
-            PKGDEST="$var_tmp" aur "${AUR_BUILD_CHROOT:-chroot--standalone}" \
-                "${chroot_args[@]}" --build "${chroot_build_args[@]}" -- "${makepkg_common_args[@]}" "${makepkg_args[@]}"
+            PKGDEST="$var_tmp" \
```

Comment:
I don't like these column fixed \ line continuations, is it really necessary to so strongly wrap longer commands?

Reply:
can you give me an example of how you would format this one (and perhaps others)?

Reply:
I wouldn't both with automated systems. It's up to alad really but I personally tend to just keep the command as long as it needs to be.

Reply:
For the time being I indented it as the line immediately below (1 line for envvars, 1 line for the rest)
Is it an acceptable tradeoff for you? Also waiting to hear from alad on this.

Reply:
I have no strong opinion about this, but the tradeoff seems fair enough.